### PR TITLE
Release 0.6

### PR DESCRIPTION
**Pull request by tieugene**
- trml2pdf.py split into executable (trml2pdf.py itself) and library
- main library module file split into separate submodules
- sources reformated and some classes/functions renamed to meet PEP8
- tests/ reordered
- some fixes for deployment

### CODE REVIEW

**Comment by romanlv** [comment]
There are only doc changes in this PR, merging from wrong branch?

File: README.md [hunk 1]
```diff
@@ -17,21 +17,22 @@ Examples
...
+import trml2pdf.doc
+print trml2pdf.doc.parse_string(file('file.rml','r').read())
 ```
  
 If you are using this for Django you can dynamically create an .rml file with the template system and then render it.
 
 ```python
 from django.template.loader import get_template
 from django.template.context import Context
+import trml2pdf.doc
```
Comment (romanlv) [inline README.md]: Introducing incompatible changes, is it possible to re-export those functions from the root `trml2pdf` so existing apps do not need to change when using new version

Comment (tieugene) [reply README.md]: You are right.
Fixed in last commit

**Review by romanlv — APPROVED** [review]
I will troubleshoot tests in different PR

**Pull request merged**